src/view/styles.ts

```diff
@@ -21,3 +22,18 @@ export const Container = styled.div`
   margin-left: 1rem;
   padding-bottom: 1.5rem;
 `;
+export const TextBeforeRendering = styled.h1`
+  font-size: 2rem;
+  font-family: "Poppins", sans-serif;
+  font-weight: 200;
+  cursor: default;
+  user-select: none;
+  margin-top: 22rem;
+`;
+
+export const ButtonClear = styled(ButtonStyle)`
```

Comment:
Maybe you can add this style to the Button component and use the type to define which style should be used